target-app/Onboarding/ViewModels/SignInViewModel.swift

```diff
@@ -13,12 +13,16 @@ protocol SignInViewModelDelegate: AuthViewModelStateDelegate {
 
 class SignInViewModel {
     
-    private var state: AuthViewModelState = .network(state: .idle) {
+    // MARK: - Observed Properties
+    @Published var state: AuthViewModelState = .network(state: .idle) {
         didSet {
             delegate?.didUpdateState(to: state)
         }
     }
     
+    // MARK: - Publishers
+    var statePublisher: Published<AuthViewModelState>.Publisher { $state }
```

Comment:
we either expose the publisher or the published var. I think this one is the only one should be public.

Reply:
that’s right! done